### ISSUE TITLE
Add a way to omit last unit

### ISSUE THREAD

**Issue by ClementCardonnel** [desktop-dart/duration#68]
Hello!
Just a quick word to say that a feature to be able to omit the last unit would be appreciated.

##### Example

```dart
// 
Duration duration = Duration(hours: 4, minutes: 30);

// 4h30min
prettyDuration(
        duration,
        abbreviated: true,
        tersity: DurationTersity.minute,
        delimiter: '',
        spacer: ''
)
```

I'd like a way to omit the last unit, in this case the minutes. Which would make `4h30`.

Thank you for considering this suggestion!

**Comment by tejainece** [comment]
Unfortunately this cannot be done currently.

I guess this only makes sense when there are only two units. What do you expect when there are also weeks in this case?

**Comment by ClementCardonnel** [comment]
Yes, I didn't envision another usage than hour and minutes. In fairness, it's not very logical to add a capability that will only be used for a specific combination. But it seems right in this specific use case. Duration for hours and minutes are often expressed that way. Or at least I think so, but maybe it's just me.
